srcpkgs/openssh/template

```diff
@@ -1,13 +1,14 @@
 # Template file for 'openssh'
 pkgname=openssh
 version=9.3p1
-revision=2
+revision=3
 build_style=gnu-configure
 configure_args="--datadir=/usr/share/openssh
  --sysconfdir=/etc/ssh --without-selinux --with-privsep-user=nobody
  --with-mantype=doc --without-rpath --with-xauth=/usr/bin/xauth
  --disable-strip --with-privsep-path=/var/chroot/ssh
- --with-default-path=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ --with-default-path=/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
```

Comment:
Why put /usr/bin before /usr/sbin, yet /sbin before /bin? IMO sbin always should go first, as on classic Unix.